hazelcast-client/src/test/java/com/hazelcast/client/MembershipListenerTest.java

```diff
@@ -124,20 +163,83 @@ public void addNullListener_thenException() throws Exception {
         return result;
     }
 
+    /**
+     * related to issue #1181
+     */
+    @Test
+    public void testAddInitialMembership_whenListenerAddedViaClientConfig() throws InterruptedException {
```

Comment:
Before the fix, It was throwing NullPointer exception according to issue mentioned in #1181 . It is testing that it does not throw any exception.